modules/storage-page-memory/src/main/java/org/apache/ignite/internal/storage/pagememory/PersistentPageMemoryTableStorage.java

```diff
@@ -113,7 +113,7 @@ protected void finishDestruction() {
     public PersistentPageMemoryMvPartitionStorage createMvPartitionStorage(int partitionId) {
         GroupPartitionId groupPartitionId = createGroupPartitionId(partitionId);
 
-        PartitionMeta meta = getOrCreatePartitionMetaOnCreatePartition(groupPartitionId);
+        StoragePartitionMeta meta = (StoragePartitionMeta) getOrCreatePartitionMetaOnCreatePartition(groupPartitionId);
```

Comment:
Let's move this cast inside `readOrCreatePartitionMeta()`, line 467:

```
return dataRegion.partitionMetaManager().readOrCreateMeta(lastCheckpointId(), groupPartitionId, filePageStore, buffer);
```

Reply:
done